### PR TITLE
11 - install.md  Install

### PR DESCRIPTION
**Pull request by halian-vilela**
Full translation of install.md 
(sorry for the last commit msg in portuguese... was in the mood)

### CODE REVIEW

File: pt_br/install.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+[http]: http://package.elm-lang.org/packages/elm-lang/http/latest
+[pipe]: http://package.elm-lang.org/packages/NoRedInk/elm-decode-pipeline/latest
+
+```bash
+elm-package install elm-lang/http
+elm-package install NoRedInk/elm-decode-pipeline
+```
+
+This will add the dependencies to your `elm-package.json` file that describes your project. (Or create it if you do not have one yet!) More information about all this [here](https://github.com/elm-lang/elm-package)!
```
Comment (martinusso) [inline pt_br/install.md]: Just remove this and :beers:

Comment (halian-vilela) [reply pt_br/install.md]: Duh! Hahaha, just left the reference there. 

Sorry guys! Done then.

File: pt_br/install.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
+> **Nota:** Se você não quiser instalar ainda, você pode acompanhar esse guia com o [editor online](http://elm-lang.org/try) e o [REPL online](http://elmrepl.cuberoot.in/).
+
+# Instalação
+
+* Mac &mdash; [instalador][mac]
+* Windows &mdash; [instalador][win]
+* Anywhere &mdash; [instalador npm ][npm] ou [gerar a partir do source][build]
```
Comment (fcevado) [inline pt_br/install.md]: > Anywhere

don't you think that is better translated?

> Qualquer S.O.

or

> Qualquer Sistema Operacional

File: pt_br/install.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+* Mac &mdash; [instalador][mac]
+* Windows &mdash; [instalador][win]
+* Anywhere &mdash; [instalador npm ][npm] ou [gerar a partir do source][build]
+
+[mac]: http://install.elm-lang.org/Elm-Platform-0.18.pkg
+[win]: http://install.elm-lang.org/Elm-Platform-0.18.exe
+[npm]: https://www.npmjs.com/package/elm
+[build]: https://github.com/elm-lang/elm-platform
+
+Após instalar por qualquer uma dessas rotas, você terá disponíveis as seguintes ferramentas em linha de comando:
```
Comment (fcevado) [inline pt_br/install.md]: > Após instalar por qualquer uma dessas rotas

sounds better translated as:

> Após instalar por qualquer uma dessas maneiras

or 

> Após instalar por qualquer uma dessas formas

File: pt_br/install.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+- [`elm-repl`](#elm-repl) &mdash; brinque com expressões em Elm
+- [`elm-reactor`](#elm-reactor) &mdash; coloque rapidamente um projeto pra rodar
+- [`elm-make`](#elm-make) &mdash; compile código Elm diretamente
+- [`elm-package`](#elm-package) &mdash; faça download de pacotes
+
+Veremos com mais detalhes como cada uma delas funciona assim que configurarmos seu editor!
+
+> **Tira-dúvidas:** O jeito mais rápido de aprender *qualquer coisa* é conversando com o pessoal da comunidade Elm. Somos amigáveis e estamos sempre felizes em ajudar!  Então, se você estiver empacado em alguma etapa da instalação ou encontrar alguma coisa estranha, viste o [Elm Slack](http://elmlang.herokuapp.com/) e faça uma pergunta. Na verdade, se você encontrar alguma coisa confusa em qualquer situação enquanto estiver aprendendo ou usando Elm, apareça e nos pergunte sobre isso. Você poderá economizar horas. Apenas venha e pergunte!
```
Comment (fcevado) [inline pt_br/install.md]: > Tira-dúvidas

sounds better translated as:

> Resolvendo Problemas

or

> Solucionando Problemas

File: pt_br/install.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+```
+Ele vai iniciar um servidor em [`http://localhost:8000`](http://localhost:8000). Você pode navegar por quaisquer arquivos Elm e ver como eles são. Dê uma olhada em `examples/01-button.elm`.
+
+**Flags importantes:**
+
+- `--port` te deixa escolher uma porta diferente da porta 8000. Ou seja, você pode escrever
+  `elm-reactor --port=8123` para fazer as coisas rodarem em `http://localhost:8123`.
+- `--address` te deixa substituir `localhost` por algum outro endereço. Por exemplo, você talvez queira utilizar `elm-reactor --address=0.0.0.0` se quiser testar um programa em Elm em um disposivo móvel conectado à sua rede local.
+
+## elm-make
```
Comment (fcevado) [inline pt_br/install.md]: All other sections on `As ferramentas em linha de comando`  has `<h3>` headers only that is `<h2>`.

I know it's like that on the original. I'll open a PR to the original repo.

**Comment by fcevado** [comment]
@halian-vilela are you still working on that?

**Comment by halian-vilela** [comment]
Yeah! Sorry for the delay, I'll make the corrections tonight!

**Pull request merged**